src/apps/chifra/pkg/tslib/tslib.go

```diff
@@ -0,0 +1,143 @@
+package tslibPkg
+
+import (
+	"encoding/binary"
+	"errors"
+	"os"
+	"sort"
+	"strings"
+	"time"
+)
+
+const tsPath = "/Users/jrush/Development/trueblocks-index/wild.0/unchained/ts.bin"
+
+type Timestamp struct {
+	Bn uint32 `json:"bn"`
+	Ts uint32 `json:"ts"`
+}
+
+var recordCount uint64 = 0
+
+func nRecords() (uint64, error) {
```

Comment:
The rest of our codebase uses comments, so that we can generate docs in the future. Should we add comments with descriptions for each of the functions in this file?

Reply:
You're right. I'll do that.